lib/components/SelectionControl/toggle-switch/toggle-switch.scss

```diff
@@ -27,7 +27,7 @@ html.dark {
     height: var(--core-size-1400);
     border-width: var(--core-borderRadius-100);
 
-    &:focus {
+    &:focus-visible {
```

Comment:
Fix behaviour onFocus for ToggleSwitcher: previously element always stayed focused after click/tap. By replacing :focus ---> :focus-visible element stayed focused onKeyDown focusing.